web/chapter_views.py

```
@@ -0,0 +1,562 @@
+        role = request.POST.get("role")
+        # Verify role is valid
+        if role in ["member", "volunteer", "co_organizer", "lead"]:
+            # If making someone else lead, current lead becomes co-organizer
+            if role == "lead" and membership.user_id != request.user.id:
+                user_membership.role = "co_organizer"
+                user_membership.save()
+            membership.role = role
+            membership.save()
+            messages.success(request, f"{membership.user.username}'s role has been updated to {role}.")
+            return redirect("manage_chapter", slug=slug)
+        else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)